### DISCUSSION TITLE
RAG with Data Access Control: Session Resources

### DISCUSSION THREAD

**Discussion by multispark** [microsoft/RAG_Hack#74] [Announcements]
This is the resources and discussion thread for this live stream: https://aka.ms/raghack/acl
It will be updated with links after the session.

Recording: https://www.youtube.com/watch?v=q27QT2IeGe8
Slides: [PPT](https://github.com/microsoft/RAG_Hack/blob/main/slides/SecuringAI_DataAccessControl.pptx)

Please ask any follow-up questions here, and the speakers will see them.

**Comment by DuboisABB** [comment]
Thanks for the great session. Matt, since you're an expert about authentication/authorization in Azure, is there any chance that you'd have a bit of time to help tackle proper token retrieval for helper scripts such as locust and the ones in the evaluator repo? Me and a few other people are struggling with this. See these two issues:
https://github.com/Azure-Samples/ai-rag-chat-evaluator/issues/97
https://github.com/Azure-Samples/azure-search-openai-demo/issues/1935
Thanks
